modules.d/90livenet/livenetroot.sh

```diff
@@ -16,6 +16,13 @@ netroot="$2"
 liveurl="${netroot#livenet:}"
 info "fetching $liveurl"
 
+if getargbool 0 'rd.writable.fsimg'; then
+
+    imgsize=$(($(curl -IL "$liveurl" | sed -n 's/Content-Length: *\([[:digit:]]*\).*/\1/p') / (1024 * 1024)))
```

Comment:
You accidentally deleted the `-s` from the `curl` call, otherwise it also prints the progress:

```
sh-5.2# curl -IL "$liveurl" | sed -n 's/Content-Length: *\([[:digit:]]*\).*/\1/p'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0  885M    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
928514048
sh-5.2# curl -sIL "$liveurl" | sed -n 's/Content-Length: *\([[:digit:]]*\).*/\1/p'
928514048
```

Other than that, the patch looks good, thanks!

Reply:
No, that was an mistake.
Thanks for spotting!
->fixed